conda_recipe/meta.yaml

```diff
@@ -4,13 +4,11 @@ package:
 
 source:
   git_url: https://github.com/ToFuProject/tofu.git
-  #git_branch: {{ environ['TRAVIS_BRANCH'] }}
   git_rev: {{ environ['REV'] }}
-  # "patches:" might be the answer for clang compilers ?
 
 build:
   script_env:
-    #   - REV
```

Comment:
Are you sure we don't need REV ? it is used just above

Reply:
it was commented, so i think no

Reply:
My mistake, ok